pkgs/games/pysolfc/default.nix

```diff
@@ -3,18 +3,23 @@
 
 buildPythonApplication rec {
   pname = "PySolFC";
-  version = "2.16.0";
+  version = "2.20.1";
 
   src = fetchzip {
     url = "https://versaweb.dl.sourceforge.net/project/pysolfc/PySolFC/PySolFC-${version}/PySolFC-${version}.tar.xz";
-    sha256 = "sha256-kklB16IrDicxqMee1kbxtoqgwcSrMjCV4HP6GtnZxo8=";
+    sha256 = "sha256-mEnsq8Su0ses+nqoSFC+Wr0MHY7aTDMbtDV8toYVNPY=";
   };
 
   cardsets = fetchzip {
     url = "https://versaweb.dl.sourceforge.net/project/pysolfc/PySolFC-Cardsets/PySolFC-Cardsets-2.1/PySolFC-Cardsets-2.1.tar.bz2";
     sha256 = "sha256-0ji6jY7zJFaaaJdInaULKUou+u934RMzYjxVDGVHbE0=";
   };
 
+  music = fetchzip {
```

Comment:
Would this change the package license?

Reply:
Not as far as I understand. The tarball containing the music has a COPYING file containing the GPL 2.0 and each track has a COPYRIGHT file in the same directory that says it's distributed under the GPL with the author's permission.

Here's an example of the latter. They all say pretty much the same thing, but differ in track name and year of copyright.

```

Astral_Dreams.it
-----------------
  Copyright (C) 1998 Carl Larsson (aka Nightbeat)
  <nightbeat@traxinspace.com>
  http://hem.passagen.se/dachande/

  Distributed under the GNU GPL by permission of the author.

```